package.json

```diff
@@ -26,6 +26,10 @@
     "url": "https://github.com/chaijs/chai/issues"
   },
   "main": "./index",
+  "exports": {
+    "require": "./index.js",
```

Comment:
if you rename this `default` and put it below `import` then there will be no warning when requiring the package, only on import

Reply:
Thanks, I didn't know that. I will push a new version tomorrow. Is this documented somewhere?

Reply:
Honestly it isn't. We only discovered that this was an issue for consumers of modules due to threads like this... and only realized this pattern worked live in the meeting on wednesday while we debating how to handle it 😅

I'm going to not have time to get any copy together in the near future but if you have time and want to send a PR I'd be happy to review and help get it landed.

Reply:
Do you mean a PR for chai or for the Node documentation or ... ? Don't really know what you mean here, because I think this PR for chai is pretty much done.

Reply:
I meant to node documentation, sorry for not being clearer. Also not at all intending this as "PRs welcome"... we'll definitely follow up with this. Just wanted to offer to help you if you had time + interest.